src/js/components/viewer/models/audio-media-object.jsx

```diff
@@ -27,6 +28,12 @@ var AudioMediaObject = React.createClass({
         this.play();
     },
 
+    // APEP if the component is unmounted we must clean up.
```

Comment:
Now because of the grid, components can be unmounted not because of themselves, but because its parent has been removed by parent.  We must clean up the player much more carefully now.  This includes all the parents children.